packages/fields/src/types/Password/Implementation.js

```diff
@@ -170,3 +171,32 @@ export class KnexPasswordInterface extends CommonPasswordInterface(KnexFieldAdap
     };
   }
 }
+
+export class PrismaPasswordInterface extends CommonPasswordInterface(PrismaFieldAdapter) {
+  constructor() {
+    super(...arguments);
+
+    // Error rather than ignoring invalid config
+    if (this.config.isUnique || this.config.isIndexed) {
+      throw (
+        `The Password field type doesn't support indexes on Knex. ` +
+        `Check the config for ${this.path} on the ${this.field.listKey} list`
+      );
+    }
+  }
+
+  getPrismaSchema() {
+    return [this._schemaField({ type: 'String' })];
+  }
+
+  getQueryConditions(dbPath) {
+    // JM: I wonder if performing a regex match here leaks any timing info that
+    // could be used to extract information about the hash.. :/
+    return {
+      // FIXME: Prisma needs to support regex matching...
```

Comment:
Is there an issue about this? I tried searching on the prisma repo but couldn't find anything

Reply:
Not that I know of, it's something I need to bring up with them, I just need to take the time to formulate the question properly so I can explain exactly what the feature would look like.